src/NUnitFramework/framework/TestContext.cs

```diff
@@ -329,9 +329,9 @@ public static void AddTestAttachment(string filePath, string description = null)
         /// </summary>
         /// <typeparam name="TSUPPORTED">The type supported by this formatter</typeparam>
         /// <param name="formatter">The ValueFormatter delegate</param>
```

Comment:
This should be updated, as the argument is not a `ValueFormatter delegate` anymore